ApprovalTests/Approvers/FileApprover.cs

```diff
@@ -52,7 +52,7 @@ public virtual ApprovalException Approve(string approvedPath, string receivedPat
                 var approvedText = File.ReadAllText(approvedPath).Replace("\r\n", "\n");
 
                 return !Compare(receivedText.ToCharArray(), approvedText.ToCharArray()) ?
-                    new ApprovalMismatchException(receivedPath, approvedPath) :
+                    new ApprovalMismatchException(receivedText, approvedText) :
```

Comment:
This is the real change. I can separate the Fody upgrade out into a separate PR if you want, but the solution will not compile with Roslyn atm